### PR TITLE
AP-4414: Amend question why new application

### PR DESCRIPTION
**Pull request by skdejong**
## What

[Amend to existing “Why are you making a new application?” question](https://dsdmoj.atlassian.net/browse/AP-4414)

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by agoldstone93 — APPROVED** [review]
Looks good to me!

File: app/views/shared/check_answers/_merits_proceeding_section.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <section class="print-no-break govuk-!-padding-top-6">
     <h2 class="govuk-heading-l"><%= proceeding.meaning %></h2>
 
-    <%= govuk_summary_list(card: { title: proceeding.meaning }, actions: !read_only) do |summary_list| %>
+    <%= govuk_summary_list(card: { title: t(".vary_order_title") }, actions: !read_only) do |summary_list| %>
```
Comment (jsugarman) [inline app/views/shared/check_answers/_merits_proceeding_section.html.erb:7]: will this not be changing the title for every proceeding?

Comment (jsugarman) [reply app/views/shared/check_answers/_merits_proceeding_section.html.erb:7]: Have test it and it does do that ^^ on the check_merits_answers question. Each of the proceedings i selected have a card title of the "vary order" title

Screenshot 2024-12-03 at 17 23 12


I believe you just need to undo this change

**Review by jsugarman — CHANGES_REQUESTED** [review]

File: app/views/shared/check_answers/_merits_proceeding_section.html.erb (hunk 1 above)
Comment (skdejong) [reply app/views/shared/check_answers/_merits_proceeding_section.html.erb]: Thanks for spotting this, @jsugarman. After discussions with you, @agoldstone93 and @kmahern, I have updated the Check Merits Answers page as shown below for now. Katharine will update the Check Merits Answers page to reflect the design in [AP-5584](https://dsdmoj.atlassian.net/browse/AP-5584).

Screenshot 2024-12-04 at 12 15 50

**Review by jsugarman — APPROVED** [review]
Great!

**Pull request merged**